client/my-sites/checkout/checkout-thank-you/google-voucher/terms-and-conditions.jsx

```diff
@@ -37,7 +37,7 @@ const googleTermsAndConditions = ( { translate } ) => (
 				{ translate(
 					'To activate this offer: Enter the promotional code in your account before %(expirationDate)s. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'To activate this offer: Enter the promotional code in your account before %(expirationDate)s. In order to participate in this offer, you must enter the code within 14 days of your first ad impression being served from your first AdWords account.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Quick and dirty? It'd be possible to update the original string

https://translate.wordpress.com/projects/wpcom/-all-translated/211031/

and replace `AdWords` with `Ads` in the translations. Fortunately `AdWords` is not translated, so it's easily found and replaced. 

_Unfortunately_ only C3PO can guarantee that we also don't require grammatical changes to the sentences because of the change. My bet is that it's unlikely - for the languages I can read it isn't - however I can't say for sure for all of them.

Reply:
I think this should be safe to merge. Our "Fuzzy bot" will catch these, and we'll s/r the existing translations once this happens (and retranslate those where we're not sure). A ping in slack once this is merged would be appreciated, to be on the safe side.

Reply:
If it's not translated would it be better to extract it as an `arg`? Would that make it simpler for translators (and easier to change next time Google update it?